HopsanCore/include/CoreUtilities/MultiThreadingUtilities.h

```diff
@@ -38,6 +38,13 @@
 #include <algorithm>
 #include "win32dll.h"
 
+#if __cplusplus >= 201103L
+#include <mutex>
+#include <chrono>
+#include <ctime>
+#include <thread>
+#endif
```

Comment:
This block of code does not seem to affect anything inside this header file. Unneeded or maybe should be placed someplace else ? (preferably inside a cpp file)

Reply:
You are right, I moved it to the cpp file.